platform/emdrv/nvm3/src/nvm3_hal_flash_ccp.c

```diff
@@ -85,6 +88,15 @@ static Ecode_t nvm3_halFlashOpen(nvm3_HalPtr_t nvmAdr, size_t flashSize) {
   Ecode_t halSta = ECODE_NVM3_ERR_NOT_OPENED;
   bool ret = ECODE_QSPI_ERROR;
 
+#if CCP_FLASH_DELAY
```

Comment:
Wonder if you have an api for such wait already. Usually, we count ticks to wait and you can event port the tick to ms values derived with your clock speed

Reply:
CCP platform provides **rsi_delay_ms**(10) api for wait functionality.
But this api is not working as we expected in our environment and we already informed to CCP platform team.
So, we are planning to use this as a temporary solution and we will fix it with proper wait API.

Reply:
do you have a ticket to track this and comeback to change it wen resolved